### PR TITLE
Mobile age chip

### PR DESCRIPTION
**Pull request by tmkrepo**
- Adds a medium sized chip to mobile child list item and a large sized one to child detailed page depicting the child's age
- Additional styling changes to enable word break in order to solve pre-existing issues in child list item spacing and responsiveness
- An additional size for the mobile icon to lower font size for 2-digit ages

### CODE REVIEW

File: frontend/src/lib-components/atoms/RoundIcon.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { FixedSpaceColumn } from 'lib-components/layout/flex-helpers'
 import { fontWeights } from '../typography'
 import { BaseProps } from '../utils'
 
-export type IconSize = 'xs' | 's' | 'm' | 'L' | 'XL' | 'XXL'
+export type IconSize = 'xs' | 's' | 'm' | 'm-age' | 'L' | 'XL' | 'XXL'
```
Comment (akheron) [inline frontend/src/lib-components/atoms/RoundIcon.tsx:18]: A new icon size should not be added for this single use case.

File: frontend/src/employee-mobile-frontend/child-attendance/ChildListItem.tsx [hunk 2]
```diff
@@ -188,6 +191,13 @@ export default React.memo(function ChildListItem({
                   />
                 </Link>
               ) : null}
+              <RoundIcon
+                content={`${childAge}v`}
+                color={
+                  childAge < 3 ? colors.accents.a6turquoise : colors.main.m1
+                }
+                size="m-age"
+              />
```
Comment (akheron) [inline frontend/src/employee-mobile-frontend/child-attendance/ChildListItem.tsx:200]: You can do it like this instead:

```ts
<AgeRoundIcon ... size="m" />
```

Bottom of the file:
```ts
const AgeRoundIcon = styled(RoundIcon)`
  &.m {
    font-size: 14px;
  }
```

The `&.m` part works around CSS selector precedence.

**Review by akheron — COMMENTED** [review]
Added few comments, otherwise looks good 👍

File: frontend/src/employee-mobile-frontend/child-attendance/ChildListItem.tsx (hunk 2 above)
Comment (tmkrepo) [reply frontend/src/employee-mobile-frontend/child-attendance/ChildListItem.tsx:200]: Age icon font size related styling extracted from base component

**Pull request merged**